platform/mbed_lib.json

```diff
@@ -128,6 +128,10 @@
         "fatal-error-auto-reboot-enabled": {
             "help": "Setting this to true enables auto-reboot on a fatal error.",
             "value": false
+        },
+        "disable-mpu": {
```

Comment:
Config options should be positive unless there's a good reason. This could be `enable-mpu`, or maybe `use-mpu` is more to the point.